src/components/leaderboard-screen/LeaderboardHighlight.tsx

```diff
@@ -0,0 +1,77 @@
+import { Medal, Trophy } from "lucide-react"
+
+import type { Doc } from "~/lib/db/types"
+import { cn } from "~/lib/utils"
+import UserAvatar from "../nav-bar/UserAvatar"
+import { Avatar } from "../ui/avatar"
+
+type LeaderboardTopThreeProps = {
+  players: [Doc<"users"> | undefined, Doc<"users"> | undefined, Doc<"users"> | undefined]
+}
+
+const PlayerCard = ({
+  player,
+  rank,
+  isGold,
+}: {
+  player: Doc<"users"> | undefined
+  rank: number
+  isGold?: boolean
+}) => {
+  if (!player) {
+    return (
+      <div
+        className={cn("flex flex-col items-center justify-center rounded-lg bg-gray-700 p-4", {
+          "h-full": isGold,
+          "h-[90%]": !isGold,
+        })}
+      >
+        <p className="text-gray-400">Player not found</p>
```

Comment:
this is a bit misleading imo - it's not that the player isn't found, there just isn't anyone playing promptracer :(